pkg/assetattributor/cloudassetinventory.go

```diff
@@ -85,15 +85,15 @@ type CloudAssetInventory struct {
 // Attribute queries the asecurityteam/asset-inventory-api service first by IP, then by hostname
 // if the first query returns no results
 func (n *CloudAssetInventory) Attribute(ctx context.Context, asset domain.NexposeAssetVulnerabilities) (domain.NexposeAttributedAssetVulnerabilities, error) {
-	if asset.LastScanned.IsZero() {
+	if asset.ScanTime.IsZero() {
```

Comment:
task:  this is where we would fall back to `asset.LastScanned` to be compatible with service prior to this one in the pipeline that is still sending `lastScanned` in the payload, yes?

Reply:
you're not wrong, but we don't use cloudassetinventory as an attributor that we use. So not including compatibility would not hurt us in this open source